### ISSUE TITLE
Scraper too narrow

### ISSUE THREAD

**Issue by ErikBoesen** [ErikBoesen/tokenwarn#5]
Not all tokens start with `Mz`. I see at least some which start with `Mj`. In addition, GitHub truncates search results to the first 100 pages.

**Comment by ioistired** [comment]
I've seen some tokens in the output that start with `Mj` and `MB`, though I can't figure out why

**Comment by ErikBoesen** [comment]
Yeah, I noticed that as well. I'm guessing the `z` just matches a different part of the token once in a while.

**Comment by ioistired** [comment]
oh perhaps

**Comment by ioistired** [comment]
So here's the token format:
![Discord token format](https://i.imgur.com/7WdehGn.png)
Basically we could brute force every possible user id as a string, encode to base64, then filter by all the ones that start with M. Search for the first two characters of those.

For example: "discord Ma Mb Mc Md Me Mf Mg Mh …"
